packages/web-ui/src/ds/molecules/DocumentTextEditor/Editor/index.tsx

```diff
@@ -20,14 +20,15 @@ import { RegularMonacoEditor } from './RegularEditor'
 export function DocumentTextEditor({
   value,
   path,
-  metadata,
```

Comment:
I was trying to find out why Monaco is slow in dev and I didn't but I saw we're just using errors from metadata so I changed this prop